### PR TITLE
Use the functions in utils_net

### PR DESCRIPTION
**Pull request by yalzhang**
Use functions in utils_net to create or delete linux bridge. And deprecate the iface-unbridge usage because it is not supported on rhel9.

Signed-off-by: Yalan Zhang <yalzhang@redhat.com>

### CODE REVIEW

**Comment by yalzhang** [comment]
run locally successfully
$avocado  run  --vt-type libvirt virtual_network.iface_bridge
JOB ID     : bd03300f1884fb16a2cc87aae25310a7ddef35ad
JOB LOG    : /root/avocado/job-results/job-2021-04-28T02.24-bd03300/job.log
 (1/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.default.shared_physical_network: PASS (112.22 s)
 (2/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.hotplug_iface.shared_physical_network: PASS (68.85 s)
 (3/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.hotplug_device.shared_physical_network: PASS (73.68 s)
 (4/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.update_with_diff_type.shared_physical_network: PASS (73.49 s)
 (5/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.multiqueues.start.shared_physical_network: PASS (56.33 s)
 (6/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.multiqueues.hotplug_device.shared_physical_network: PASS (74.06 s)
RESULTS    : PASS 6 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 460.61 s

**Comment by yalzhang** [comment]
depends on https://github.com/avocado-framework/avocado-vt/pull/3035

**Review by chunfuwen — APPROVED** [review]
The change looks good to me

File: libvirt/tests/src/virtual_network/iface_bridge.py [hunk 1]
```diff
@@ -166,7 +143,9 @@ def check_net_functions(guest_ip, ping_count, ping_timeout, guest_session, host_
     mac = utils_net.generate_mac_address_simple()
 
     try:
-        create_bridge(bridge_name, iface_name)
+        if libvirt.check_iface(bridge_name, "exists", "--all"):
+            test.cancel("The bridge %s already exist" % bridge_name)
+        utils_net.create_linux_bridge(bridge_name, iface_name)
```
Comment (Yingshun) [inline libvirt/tests/src/virtual_network/iface_bridge.py:148]: I think you should test the status of utils_net.create_linux_bridge which calls utils_misc.cmd_status_output to check if the bridge is created correctly or not.

Here is an example of a command that failed to execute.
>>> utils_misc.cmd_status_output("ssss", shell=True, verbose=True, ignore_status=False)
(1, "Command 'ssss' failed.\nstdout: b''\nstderr: b'/bin/sh: ssss: command not found\\n'\nadditional_info: None")

**Comment by yalzhang** [comment]
run the cases again as the function changes in the utils_net
$ avocado run --vt-type libvirt  virtual_network.iface_bridge
JOB ID     : 5b2995eec44aa921bc3bd4ecbc3577a88289bdbc
JOB LOG    : /root/avocado/job-results/job-2021-05-11T23.35-5b2995e/job.log
 (1/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.default.shared_physical_network: PASS (121.16 s)
 (2/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.hotplug_iface.shared_physical_network: PASS (64.39 s)
 (3/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.hotplug_device.shared_physical_network: PASS (71.28 s)
 (4/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.update_with_diff_type.shared_physical_network: PASS (74.47 s)
 (5/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.multiqueues.start.shared_physical_network: PASS (56.78 s)
 (6/6) type_specific.io-github-autotest-libvirt.virtual_network.iface_bridge.multiqueues.hotplug_device.shared_physical_network: PASS (72.04 s)
RESULTS    : PASS 6 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 462.11 s

**Comment by yalzhang** [comment]
@Yingshun  Please help to review, Thank you!

**Review by Yingshun — APPROVED** [review]

File: libvirt/tests/src/virtual_network/iface_bridge.py (hunk 1 above)
Comment (yalzhang) [reply libvirt/tests/src/virtual_network/iface_bridge.py]: Updated, Thank you!

**Pull request merged**